src/main/java/com/thealgorithms/ciphers/MonoAlphabetic.java

```diff
@@ -0,0 +1,44 @@
+package com.thealgorithms.ciphers;
+
+public final class MonoAlphabetic {
+
+    // Private constructor to prevent instantiation of utility class
+    private MonoAlphabetic() {
+        throw new UnsupportedOperationException("Utility class");
+    }
+
+    // Encryption method
+    public static String encrypt(String data, String key) {
+        StringBuilder sb = new StringBuilder();
+        data = data.toUpperCase();
+
+        for (char c : data.toCharArray()) {
+            if (c >= 'A' && c <= 'Z') {
+                int idx = c - 'A'; // Index in alphabet
```

Comment:
Consider adding two methods: `charToPos` and `posToChar`.

Reply:
Added the methods charToPos and posToChar.